aea/registries/base.py

```diff
@@ -341,7 +341,19 @@ def setup(self) -> None:
         :return: None
         """
         for item in self.fetch_all():
-            item.setup()
```

Comment:
we now need to skip the setup of skill components whose skill is deactivated, since a skill can be deactivated in the `__init__` (hence at load time).